src/renderer/components/settings-general-package-author.tsx

```diff
@@ -0,0 +1,73 @@
+import { Callout, FormGroup, InputGroup } from '@blueprintjs/core';
+import { observer } from 'mobx-react';
+import * as React from 'react';
+
+import { AppState } from '../state';
+
+interface PackageAuthorSettingsProps {
+  appState: AppState;
+}
+
+interface IPackageAuthorSettingsState {
+  value: string;
+}
+
+/**
+ * Settings package.json author info
+ *
+ * @class PackageAuthorSettings
+ * @extends {React.Component<PackageAuthorSettingsProps, IPackageAuthorSettingsState>}
+ */
+@observer
+export class PackageAuthorSettings extends React.Component<
+  PackageAuthorSettingsProps,
+  IPackageAuthorSettingsState
+> {
+  constructor(props: PackageAuthorSettingsProps) {
+    super(props);
+
+    this.state = {
+      value: this.props.appState.packageAuthor,
+    };
+
+    this.handlePackageAuthorChange = this.handlePackageAuthorChange.bind(this);
+  }
+
+  /**
+   * Set the author information in package.json when processing uploads to gist
+   *
+   * @param {React.ChangeEvent<HTMLInputElement>} event
+   */
+  public handlePackageAuthorChange(event: React.FormEvent<HTMLInputElement>) {
+    const { value } = event.currentTarget;
+
+    this.setState({
+      value,
+    });
+
+    this.props.appState.packageAuthor = value;
+  }
+
+  public render() {
+    const { packageAuthor } = this.props.appState;
+
+    const packageAuthorLabel = 'Set package.json author field';
+
+    return (
+      <div>
+        <h4>Package Author</h4>
+        <Callout>
+          <FormGroup label={packageAuthorLabel}>
+            <InputGroup
+              placeholder={packageAuthor}
```

Comment:
Since `this.props.appState.packageAuthor` is tied to `this.state.value`, it seems like the placeholder would never show?
```suggestion
```